CODE_OF_CONDUCT.md

```diff
@@ -55,11 +55,11 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at . All complaints will be reviewed and
-investigated and will result in a response that is deemed necessary and
-appropriate to the circumstances. The project team is obligated to maintain
-confidentiality with regard to the reporter of an incident. Further details of
-specific enforcement policies may be posted separately.
+reported by contacting the project team at (TODO: email?). All complaints will
```

Comment:
Need to know what to put here.

Reply:
For now, you can put segun.adebayo@chakra-ui.com.

Thanks for this!